design-documents/graph-ql/coverage/negotiableQuotes.graphqls

```diff
@@ -0,0 +1,361 @@
+
+    createOrderWithGiftCardForNegotiableQuote(
+        giftCardAccountData: NegotiableQuoteGiftCardAccountDataInput!
+    ): CreateOrderWithGiftCardForNegotiableQuoteOutput! @doc(description:"Assigns GiftCard information and cart ID for the negotiable quote to create order")
+
+    removeGiftCardFromNegotiableQuote(
+        giftCardsData: NegotiableQuotegiftCardsInput!
+    ): RemoveGiftCardFromNegotiableQuoteOutput! @doc(description:"Removes a gift card that has been applied to a negotiable quote")
+}
+
+type Customer {
+        negotiable_quotes(
+        page_size: Int = 20,
+        current_page: Int = 1,
+        filter: NegotiableQuoteFilterInput
+    ): [NegotiableQuote] @doc(description: "Get negotiable quotes of a customer")
+}
+
+type NegotiableQuote {
+    billing_address: BillingAddress! @doc(description: "Billing address of negotiable quote")
+    attachements: [AttachmentContent] @doc(description: "Negotiable quote Attachements")
+    comments: [NegotiableQuoteComment] @doc(description: "Negotiable quote comments")
+    totals: TotalInfo @doc(description: "Negotiable quote totals info.")
+    payment_info: NegotiableQuotePaymentInfo! @doc(description: "Negotiable quote payment info.")
+}
+
+input NegotiableQuoteFilterInput {
+    ids: FilterEqualTypeInput @doc(description: "Filter Customer Negotibale quotes with an negotiable quote ID or list of negotiable quote IDs")
+    name: FilterMatchTypeInput @doc(description: "Filter by display name of the negotiable quote")
+}
+
+type BillingAddress {
+    id: ID! @doc(description: "The ID of the billing address")
+    firstname: String @doc(description: "The first name of the person associated with the billing address")
+    lastname: String @doc(description: "The family name of the person associated with the billing address")
+    customer_id: String @doc(description: "The customer ID")
+    email: String @doc(description: "The customer email id")
+    company: String @doc(description: "The customer's company")
+    telephone: String @doc(description: "The telephone number")
+    street: [String] @doc(description: "An array of strings that define the street number and name")
+    city: String @doc(description: "The city or town")
+    region: String @doc(description: "The region name")
+    region_id: String @doc(description: "The region ID")
+    region_code: String @doc(description: "The region code")
+    postcode: String @doc(description: "The customer's ZIP or postal code")
+    country_id: String @doc(description: "Deprecated: use `country_code` instead.")
+    same_as_billing: Int @doc(description: "Billing address confirmation")
+    save_in_address_book: Int @doc(description: "Save billing address in address book")
+}
+
+type AttachmentContent @doc(description: "Negotiable quote attachment file") {
+    base64_encoded_data: String
```

Comment:
It looks like we need to follow the same approach here as discussed in another PR: https://github.com/magento/architecture/pull/378/files#r428196235

Reply:
This URL is not pointing to any specific location can you please tell us which conversation you are referring to. Thanks.

Reply:
https://github.com/magento/architecture/pull/378#discussion_r428196235

Reply:
In https://github.com/magento/architecture/pull/378#discussion_r428196235 they have removed the Attachmentcontent as they have not finalized the approach, can we remove that here also? @paliarush @mbrinton01 @DrewML 
Thanks.